src/components/email-list/EmailList.tsx

```diff
@@ -27,6 +27,7 @@ export interface EmailListState {
 class _EmailList extends React.Component<EmailListProps, EmailListState> {
   stateOptions: Array<ReactSelectOption>;
   selectStyles = {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
     option: (provided: any) => ({
```

Comment:
in this case, `provided` can be `React.CSSProperties` (and with fully explicit typing, selectStyles' type contains an option property typed `(provided: React.CSSProperties) => React.CSSProperties`-- and if it didn't an anonymous type with an index signature (such as `{[propName: string]: string | {[propName: string]: string}}`, even) should be preferred over disabling settings like this

Reply:
thank you so much! didn't know any of that before so thank you for teaching me!